### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by kryvets-vadym**
[DEMO LINK](https://kryvets-vadym.github.io/react_goods-selector/)

### CODE REVIEW

**Comment by AndreyChernysh** [comment]
Nothing to add good job!

**Review by AndreyChernysh — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +17,89 @@ export const goods = [
...
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  enableSelectionOnClick = (product: string) => {
```
Comment (Olexandr6) [inline src/App.tsx:29]:
```suggestion
  handleProductSelection = (product: string) => {
```

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +17,89 @@ export const goods = [
...
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  disableSelectionOnClick = () => {
```
Comment (Olexandr6) [inline src/App.tsx:33]:
```suggestion
  handleProductRemoving = () => {
```

File: src/App.tsx [hunk 3]
```diff
@@ -15,71 +17,89 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedProduct: string | null,
```
Comment (Olexandr6) [inline src/App.tsx:21]: You don't need `null` here.  You can use '' empty string as default value

File: src/App.tsx [hunk 4]
```diff
@@ -15,71 +17,89 @@ export const goods = [
...
+          <tbody>
+            {goods.map(good => (
+              <tr
+                data-cy="Good"
+                className={classNames({
+                  'has-background-success-light': good === selectedProduct,
+                })}
+              >
+                <td>
+                  {selectedProduct === good
```
Comment (Olexandr6) [inline src/App.tsx:72]: Let's move the expression to a variable or a function

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
Almost done👍 Now, let's work on improvements:

**Review by Olexandr6 — APPROVED** [review]

**Pull request closed without merging**